src/NHibernate/Collection/Generic/PersistentGenericBag.cs

```diff
@@ -46,6 +49,8 @@ public class PersistentGenericBag<T> : AbstractPersistentCollection, IList<T>, I
 		 */
 		private IList<T> _gbag;
 
+		private IQueryable<T> queryable;
```

Comment:
It's a hundreds years old PR, so I don't remember/know why I didn't put underscore in front of the field name.,